src/components/Trade.js

```diff
@@ -1,37 +1,112 @@
+    console.log(choice);    // to be removed
+  }
+
   return (
     <main className={classes.content}>
       <div className={classes.appBarSpacer} />
       <Container maxWidth="lg" className={classes.container}>
-        <Grid container spacing={3}>
+        <Grid container item spacing={3} alignItems="stretch">
           <Grid item xs={12}>
-            <Paper>
-              <CanvasJSCandlestickChart />
-            </Paper>
+            <Card>
+              <CardContent>
+                <div className="chart-header">
+                  <Typography variant="h2" color="primary">
+                    STOCK CHART
+                  </Typography>
+                  <Button
+                    aria-controls="fade-menu"
+                    aria-haspopup="true"
+                    onClick={handleClick}
+                  >
+                    {chartType}
+                  </Button>
+                  <Menu
+                    id="fade-menu"
+                    anchorEl={anchorEl}
+                    keepMounted
+                    open={open}
+                    TransitionComponent={Fade}
+                  >
+                    <MenuItem onClick={() => handleClose("CANDLESTICK")}>
+                      CANDLESTICK
+                    </MenuItem>
+                    <MenuItem onClick={() => handleClose("SPLINE AREA")}>
+                      SPLINE AREA
+                    </MenuItem>
+                  </Menu>
+                </div>
+                <div className="chart-content">
+                  {chartType === "CANDLESTICK" ? (
+                    <CandlestickChart>{option}</CandlestickChart>
+                  ) : (
+                    <SplineAreaChart>{option}</SplineAreaChart>
+                  )}
+                </div>
+              </CardContent>
+            </Card>
           </Grid>
           <Grid item xs={4}>
-            <Paper className={fixedHeightPaper}>
-              <Deposits />
-            </Paper>
+            <Card>
+              <CardContent>
+                <CompanyOverview>{option}</CompanyOverview>
+              </CardContent>
+            </Card>
           </Grid>
           <Grid item xs={4}>
-            <Paper className={fixedHeightPaper}>
-              <Deposits />
-            </Paper>
+            <Card>
+              <CardContent>
+                <PaperTrading />
+              </CardContent>
+            </Card>
           </Grid>
           <Grid item xs={4}>
-            <Paper className={fixedHeightPaper}>
-              <Deposits />
-            </Paper>
+            <Card>
+              <CardContent>
+                <Typography variant="h2" color="primary">
+                  MY WATCHLIST
+                </Typography>
+                <List>
+                  <Watchlist onClick={handleOption}>AMZN</Watchlist>
+                  <Watchlist onClick={handleOption}>AAPL</Watchlist>
+                  <Watchlist onClick={handleOption}>TSLA</Watchlist>
```

Comment:
Not sure if this is the correct way to implement <code>onClick</code> in order for the chart to render different companies. Currently, the <code>option</code> state does changes correctly based on different clicks, and this can be verified with <code>console.log(option)</code>. However, the stock chart does not seem to automatically update when the <code>option</code> value changes.

Reply:
I'm not sure why it's not rendering. Probably you might want to try to handle the `onClick` event here, instead of on `<WatchList>`. This makes the code easier to read also.

Reply:
Still doesn't work as of commit <code>b221aa8</code>. We will just open this issue later.